docs/cryptodoc/src/05_09_frodokem.rst

```diff
@@ -0,0 +1,298 @@
+.. _pubkey/frodokem:
+
+FrodoKEM
+=================
+
+Botan's FrodoKEM implementation is found in
+:srcref:`src/lib/pubkey/frodokem/` and follows [FrodoKEM-ISO]_.
+The parameter sets shown in the tables below are supported.
+
+.. _pubkey/frodokem/parameter_table:
+
+.. table::  Supported FrodoKEM parameter sets (see Tables A.1 and A.2 of [FrodoKEM-ISO]_). ``<PRG>`` can either be ``AES`` or ``SHAKE``, depending on whether AES-128 or SHAKE-128 is used for expanding the seed for the matrix :math:`A`.
```

Comment:
PRG is not defined anywhere in the documentation. Would it be possible to include that it means pseudorandom generator somewhere?

Reply:
I renamed it to DRBG which is perhaps a more common abbreviation.